src/main/java/com/medical/doctor/controller/DoctorController.java

```diff
@@ -310,4 +306,36 @@ public String addExpertisation(@PathVariable Integer expertise) {
 		}
 	}
 
+	@RequestMapping(method = RequestMethod.PUT, produces = MediaType.APPLICATION_JSON_VALUE, value = "/signup")
+	@ResponseBody
+	public Integer doctorSignUp(@RequestBody Doctor doctor) {
```

Comment:
@aviral-92  Entity Object using directly change it to Request Object and map request object into Entity Object